src/test/java/igrad/logic/parser/CourseBookParserTest.java

```diff
@@ -0,0 +1,82 @@
+package igrad.logic.parser;
+
+import static igrad.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static igrad.commons.core.Messages.MESSAGE_UNKNOWN_COMMAND;
+import static igrad.testutil.Assert.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
```

Comment:
same: static imports, in particular, this had been removed;
`import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;`